ultras-io/ultras
# Initialize monorepo, create some packages, test usage

## Usage example
```js
const { multiply } = require("@ultras/shared");

console.log(multiply(1, 2));
```
## To test
> node packages/cli/index.js 

@hay92k please go through the  [README](https://github.com/ultras-io/ultras/blob/configuration/intialize-monorepo/packages/cli/README.md) to see how we can use the cli
